### DISCUSSION TITLE
AdvancedBloomFilter performance is not good?

### DISCUSSION THREAD

**Discussion by kringt06** [pixijs/filters#218] [Q&A]
When I use [AdvancedBloomFilter], I lose frames and the device burns.

```js
new AdvancedBloomFilter({
      threshold: 0.4,
      bloomScale: 3,
      blur: 40,
      quality: 20
});


_update -> 

alpha  ?:  0 ~ 1;

AdvancedBloomFilter.threshold = alpha > 0.4 ? alpha : 0.4

```

Is there anything you can do to improve performance?

**Comment by ivanpopelyshev** [comment]
Yes, filters are like that.

1. try `filter.resolution = 0.5` and see how it affects quality.

There are more effective ways for blur 

2. Decrease params that are for KawaseBlurFilter.

https://github.com/pixijs/pixi-filters/blob/master/filters/advanced-bloom/src/AdvancedBloomFilter.js#L73

Your current quality and blur are too big.

3. Ask me to add more effective blur for that filter, but it can take a loooong time because my queue on heavy-difficult tasks is big.
